.gitignore

```diff
@@ -5,3 +5,4 @@ llvm_ir_dataset_utils.egg-info/
 *.tar
 *.sif
 *.swp
+*~
```

Comment:
This seems unrelated?

Also, where are these coming from?

Reply:
These are undo and backup file suffixes which are created by vim.